src/sep6/WithdrawProvider.ts

```diff
@@ -0,0 +1,65 @@
+import queryString from "query-string";
+
+import { Omit } from "../util";
+import { fetchKycInBrowser } from "./fetchKycInBrowser";
+import { TransferProvider } from "./TransferProvider";
+import {
+  FeeArgs,
+  InteractiveKycNeeded,
+  TransferResponse,
+  WithdrawRequest,
+} from "./types";
+
+export class WithdrawProvider extends TransferProvider {
+  public async withdraw(args: WithdrawRequest): Promise<TransferResponse> {
+    const search = queryString.stringify(args);
+    const response = await fetch(`${this.transferServer}/withdraw?${search}`);
+    return response.json();
+  }
+
+  public async fetchSupportedAssets() {
+    const { withdraw } = await this.fetchInfo();
+    return withdraw;
+  }
+
+  public async fetchFinalFee(args: Omit<FeeArgs, "operation">) {
+    return super.fetchFinalFee({
+      ...args,
+      operation: "withdraw",
+    });
+  }
+
+  public async fetchKycInBrowser({
+    response,
+    request,
+    window: windowContext,
+  }: {
+    response: InteractiveKycNeeded;
```

Comment:
Looking at this without knowing intimately how the system works, I wouldn't know what shape "response" was supposed to be. I would have guessed `InteractiveKycNeeded` was a boolean. So maybe the name can give clearer hints to what this actually is?